app/components/view-data-text/macro.njk

```diff
@@ -0,0 +1,5 @@
+{% macro viewDataText(params) %}
+  {% if params.data %}
+    <div data-test-id="{{ params.dataTestIdIdentifier }}" class="bc-c-data-text {{ params.classes }}">{{ params.data }}</div>
```

Comment:
do id and identifier mean the same thing? so it is like data test identifier identifier

Reply:
Yeah, we have a separate task in devops to change this https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_sprints/taskboard/Buying%20Catalogue%20Team/Buying%20Catalogue/Iteration%2015?workitem=4280

Reply:
😆

Reply:
It could potentially break the marketing pages so leaving it until afterwards

Reply:
Katie that was one of your naming things in Tag component. We went with what you had done there 😂 but we agree it needs changing.

Reply:
It's hard to obtain perfection when you are leading the revolution

Reply:
You laughing at your own joke!!! 😆

Reply:
Thumbs down for the self react

Reply:
I couldn't be bothered editing the comment so 👎 👎 to you both